person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @age = age
+    @parent_permission = parent_permission
+    @name = name
+  end
+
+  def of_age?
+    @age >= 10
+  end
```

Comment:
- [ ] It would be better if the `of_age` method is a private method and the `can_use_services` method is a public method.